client/common/sass/components/_incident-index.sass

```diff
@@ -1,2 +1,332 @@
+				content: url(../icons/arrow-down-white.svg)
+
+.incident-index__download-select
+	position: absolute
+	padding: 0
+	display: flex
+	flex-direction: column
+	z-index: 100
+	left: 0
+	right: 0
+
+	+tablet-up
+		left: auto
+
+.incident-index__download-wrapper
+	display: flex
+
+	+tablet-up
+		width: 20rem
+
+.incident-index__download-list
+	background: $black
+	position: relative
+	flex: auto
+
+.incident-index__download-item
+	box-sizing: border-box
+	display: flex
+	justify-content: space-between
+	width: 100%
+	padding: 0.5rem 0.75rem
+	color: $white
+	text-decoration: none
+
+	&--more
+		text-decoration: underline
+
+.incident-index__incident-count
+	color: $gray-text
+
+.incident-index__download-button
+	font-weight: 700
+	width: calc(100% - 1.5rem - 6px)
+	text-align: center
+	border: 3px solid
+	border-color: $black
+
+	&:focus
+		border-width: 6px
+
+	&[disabled]
+		color: $tag-inactive
+		border-color: $tag-inactive
+
+	+tablet-up
+		width: auto
+		border: none
+		// the calc here makes space on right edge for the down-arrow symbol
+		// which is 16px (inner-width) + 1.5rem (padding) + 3px (border) wide
+		padding: calc(0.5rem + 3px) calc(1rem + 1.5rem + 19px) calc(0.5rem + 3px) 0.5rem
+		transition: color 0.01s ease-in 0.08s
+
+		&::after
+			content: url(../icons/arrow-down.svg)
+			height: 16px
+			width: 24px
+			border: 3px solid
+			border-color: $black
+			position: absolute
+			right: 0px
+			top: 0px
+			padding: 0.75rem 0.5rem
+			//padding left of parent
+			transition: content 0.01s ease-in 0.08s
+
+		&:focus
+			//margin: -3px 0
+			//border-width: 6px 3px 6px 6px
+
+			&::after
+				border-width: 6px
+				right: -3px
+				top: -3px
+				//box-shadow: rgba(9, 105, 218, 0.3) 0px 0px 0px 3px
+
+		&[disabled]
+			color: $tag-inactive
+			border-color: $tag-inactive
+
+			&::after
+				content: url(../icons/arrow-down-inactive.svg)
+				border-color: $tag-inactive
+
+.incident-index__filters-mobile-controls
+	display: none
+
+.incident-index__filters
+	background: $section-background
+	box-sizing: border-box
+	margin-top: 4rem
+	position: fixed
+	z-index: 600
+	inset: 0
+	max-width: 0
+	overflow: hidden
+	transition: max-width 350ms ease-out
+
+	&[data-visible='true']
+		max-width: 100vw
+		display: flex
+		flex-direction: column
+
+		.filters__header
+			display: none
+
+		.incident-index__filters-mobile-controls
+			background-color: $white
+			padding: 2rem
+			display: flex
+			justify-content: space-between
+			align-items: center
+			z-index: 610
+
+		.incident-index__filters-wrapper
+			overflow: auto
+			flex: 1
+
+	+tablet-up
+		grid-column: 1 / 4
+		grid-row: span 2
+		align-self: start
+		margin-top: 0
+		position: static
+		max-width: none
+		max-height: calc(100vh - 120px)
+		overflow-y: scroll
+		position: sticky
+		top: 0
+
+.incident-index__results
+	grid-column: 1 / -1
+
+	+tablet-up
+		grid-column: 4 / -1
+		display: grid
+		+grid-columns(9, 2rem)
+		position: sticky
+		top: 0
+		overflow-y: scroll
```

Comment:
I think the sticky and scroll logic here needs some adjusting based on the figma design. I don't think the entire result section needs to be sticky, but the heading of the result, and the count, sort, etc are what's sticky based on [this figma presentation](https://www.figma.com/proto/H3uTXxT88ofhdebUFgzEo9/PFT-%E2%80%93-Hand-off-Guide?node-id=4474%3A116770&scaling=min-zoom&page-id=3897%3A59348&starting-point-node-id=3897%3A59349)